docs/machine-learning/ml-tpm-guidance.md

```diff
@@ -0,0 +1,60 @@
+# The role of a TPM in AI projects?
+
+In this document, we explore some of the Program Management considerations for Machine Learning (ML) projects and suggest recommendations for Technical Program Managers (TPM) to effectively work with Data engineering teams.
+
+## Is it an AI problem?
+
+In Artificial Intelligence (AI) projects, the ML component is generally a part of an overall business problem and **NOT** the problem itself. Few considerations for identifying the right fit for your project:
+
+- Determine the value for Machine Learning within the larger scope of a project. Think about the overall business problem first and then evaluate if ML can help address a part of the problem space.
+
+- Engage experts in human experience and employ techniques such as [Design Thinking](https://www.ideou.com/blogs/inspiration/what-is-design-thinking) to **understand the customer needs** and human behavior first. For example, if your customer wants to talk about [Topic Modeling](https://en.wikipedia.org/wiki/Topic_model) instead of [Personas](https://en.wikipedia.org/wiki/Persona_(user_experience)), you are having the wrong conversation.
+
+- Focus on **System Design** principles to identify the architectural components, entities, interfaces, constraints. Ask the right questions early and explore design alternatives with the engineering team.
+
+- Think hard about the **costs of ML** and whether you can solve a repetitive problem at scale. Many a times, you can solve customer problems with data analytics, dashboards, or rule-based algorithms.
+
+### Taming the ambiguity beast
+
+ML projects can be plagued with a phenomenon we can call as the "**Death by Unknowns**". Unlike software engineering projects, ML focused projects can result in quick success early (aka sudden decrease in error rate), but this may flatten eventually. Few things to consider:
+
+- **Set clear expectations**: Identify the performance metrics and discuss on a "good enough" prediction rate that will bring value to the business. An 80% "good enough" rate may save business costs and increase productivity but if going from 80 to 95% would require unimaginable amount of cost and effort. Is it worth it? Can it be a progressive road map?
+  
+- Create a smaller team and **undertake a feasibility analysis** through techniques like [EDA](https://en.wikipedia.org/wiki/Exploratory_data_analysis) (Exploratory Data Analysis). A [feasibility study](https://microsoft.github.io/code-with-engineering-playbook/machine-learning/ml-feasibility-study/) is much cheaper to evaluate data quality, customer constraints and model feasibility. It allows a TPM to better understand customer use cases and current environment and can act as a fail-fast mechanism. Note that feasibility should be shorter (in weeks) else it misses the point of saving costs.
```

Comment:
```suggestion
- Create a smaller team and **undertake a feasibility analysis** through techniques like [EDA](https://en.wikipedia.org/wiki/Exploratory_data_analysis) (Exploratory Data Analysis). A [feasibility study](ml-feasibility-study.md) is much cheaper to evaluate data quality, customer constraints and model feasibility. It allows a TPM to better understand customer use cases and current environment and can act as a fail-fast mechanism. Note that feasibility should be shorter (in weeks) else it misses the point of saving costs.
```
The playbook's website will generate the link based on relative reference.

Reply:
fixed to relative reference